server/src/Core/Specifications/GameNameDuplicationsSpecification.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Ardalis.Specification;
+using BGS.ApplicationCore.Entities;
+
+namespace BGS.ApplicationCore.Specifications;
+
+public sealed class GameNameDuplicationsSpecification : SingleResultSpecification<Game, bool>
```

Comment:
This specification is placed in the core module. Let's rename it to a more "common" name `GameByNameSpecification`.